test/watcher.js

```diff
@@ -18,9 +18,9 @@ describe('watcher', function() {
     await rimraf(__dirname + '/input');
   });
 
-  afterEach(function() {
+  afterEach(async function() {
     if (b) {
-      b.stop();
+      await b.stop();
```

Comment:
Apparently the watcher tests, didn't wait for workers and watcher to properly shutdown